### PR TITLE
plug 2.0.14

### PR DESCRIPTION
**Pull request by aren55555**

### CODE REVIEW

File: Casks/plug.rb [hunk 1]
```diff
@@ -4,11 +4,11 @@
...
     url "https://plugformac.com/files/Plug-#{version}.dmg"
   else
-    version '2.0.11'
-    sha256 '9542bbae3beadca999246012cc5ede97ae6e992fdd622c482f1c1bf0145a7f8d'
+    version '2.0.14'
     appcast "https://www.plugformac.com/updates/plug#{version.major}/sparklecast.xml",
-            checkpoint: '9bf2e94dd7486633b9dd622d7dbc631f346ec6304744ec157da2a476886e9b03'
+            checkpoint: '46430df9121b04a98a6ccb058ebbd97e4e396760c3cf70e5902b2c4acf95f88b'
     url 'https://www.plugformac.com/updates/plug2/Plug-latest.dmg'
+    sha256 'afab04697b729bf6fda4042b2292c2c84803d5d1cdf18a491d4025e699403240'
```
Comment (reitermarkus) [inline Casks/plug.rb]: This should stay where it was, below `version`.

File: Casks/plug.rb [hunk 2]
```diff
@@ -4,11 +4,11 @@
     sha256 'd8eed07bd1f84d6f1daa7a0699617f2e21c01df2e68924945bcb4889d1251d01'
     url "https://plugformac.com/files/Plug-#{version}.dmg"
   else
-    version '2.0.11'
-    sha256 '9542bbae3beadca999246012cc5ede97ae6e992fdd622c482f1c1bf0145a7f8d'
+    version '2.0.14'
     appcast "https://www.plugformac.com/updates/plug#{version.major}/sparklecast.xml",
```
Comment (reitermarkus) [inline Casks/plug.rb]: `appcast` should be below the `url` stanza.

**Comment by vitorgalvao** [comment]
Please read the instructions. You are not adding a new cask, and you didn’t delete what you don’t need. It looks like you just checked the boxes without performing the actions. When that’s the case, it makes our work harder.

If, alternatively, the instructions were unclear, please say so, so we can improve them.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in b98e328fc6e63227270b5107476ee2ea1a851233. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**